ThunderCloud/LocalisationController.swift

```diff
@@ -445,7 +449,7 @@ public class LocalisationController: NSObject {
 			
 			// If any of the localised values (One for each language) has been edited
 			if localisation.localisationValues.first(where: { (localisationKeyValue) -> Bool in
-				return localisationKeyValue.localisedString == "".localised(with: localisationKey)
+				return localisationKeyValue.localisedString != "".localised(with: localisationKey)
```

Comment:
Is this intended?

Reply:
Yes, if `localisationKeyValue.localisedString != "".localised(with: localisationKey)` then we know the localisation has been edited and therefore colour the highlighted UI amber!